### PR TITLE
SPECT and synergistic PET/SPECT fixes

### PR DESCRIPTION
**Pull request by samdporter**
Synergistic & SPECT SIRF Exercises updated.  
nbstripout run on Synergistic, SPECT & Geometry notebooks

### CODE REVIEW

**Comment by DANAJK** [comment]
I definitely manually deleted the "id" lines from Geometry notebooks a_ b_ and c_
Now I see there is a new "id" tag in the files that are on Git master and in this pull request
Not sure how they got there - perhaps inserted by nbstripout???

**Comment by DANAJK** [comment]
> not sure what happened here but @samdporter PR should not modify Geometry files of course. (I certainly didn't)



> not sure what happened here but @samdporter PR should not modify Geometry files of course. (I certainly didn't)

I think Sam's PR just changed some other little bit of text in the nb, not the "id" part?

**Comment by samdporter** [comment]
Got rid of my changes. Sorry about that

File: notebooks/SPECT/SPECT_OSEM.ipynb [hunk 1]
```diff
@@ -29,6 +22,14 @@
     "OSEM is (still) the most common algorithm in use in clinical PET."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is a skeleleton for a simple OSEM reconstruction using the SIRF SPECT projector and OSEM reconstructor.\n",
```
Comment (KrisThielemans) [inline notebooks/SPECT/SPECT_OSEM.ipynb:29]:
```suggestion
    "This notebook is a skeleton for a simple OSEM reconstruction using the SIRF SPECT projector and OSEM reconstructor.\n",
```

File: notebooks/SPECT/SPECT_OSEM.ipynb [hunk 2]
```diff
@@ -265,7 +255,8 @@
    "source": [
     "# create initialisation image and set up reconstructor\n",
     "init_image = make_cylindrical_FOV(image.get_uniform_copy(1))\n",
-    "OSEM_reconstructor.set_up(init_image)"
+    "\n",
+    "### Now set_up the reconstructor object using the intial image ###"
```
Comment (KrisThielemans) [inline notebooks/SPECT/SPECT_OSEM.ipynb:259]:
```suggestion
    "### Now set_up the reconstructor object using the initial image ###"
```

File: notebooks/Synergistic/cil_joint_tv_PET_SPECT.ipynb [hunk 3]
```diff
@@ -65,7 +55,8 @@
    "outputs": [],
    "source": [
     "# Make sure figures appears inline\n",
-    "%matplotlib notebook"
+    "%matplotlib notebook\n",
+    "%matplotlin inline"
```
Comment (KrisThielemans) [inline notebooks/Synergistic/cil_joint_tv_PET_SPECT.ipynb:59]:
```suggestion
    "%matplotlib inline"
```

**Review by KrisThielemans — CHANGES_REQUESTED** [review]

**Pull request merged**